content/enterprise/before-installing/index.mdx

```diff
@@ -132,7 +132,9 @@ Terraform Enterprise requires **at least one** of the following Docker Engine co
 
 1. Docker Engine 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x with `runc` v1.0.0-rc93 or greater.
 1. Docker Engine 20.10.x with `libseccomp` 2.4.4 or greater.
-1. Docker Engine 1.13.1 (RHEL only), 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x using a modified `libseccomp` profile.
+1. 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x using a [modified `libseccomp` profile](https://www.terraform.io/enterprise/before-installing#option-2-docker-engine-with-a-compatible-libseccomp-version).
```

Comment:
I see you removed "Docker Engine" which I agree with since we specify that they are all docker engines above. Can we remove that from the first two bullets, too? also! Is this a repeat of the first option?

Reply:
I'll make that update. And no, it's different from 1. 1 is any of those version combined with the specified version of runc. The other one is if you don't have that updated runc installed, you instead need the modified libseccomp profile. My brain hasn't come up with a better way to display this data, but I am open to suggestions.

Reply:
okay I'll leave as is for now :)